_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -3439,7 +3439,7 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
                     }
 
                     changed = true;
-                    par.calcParam.targetKbps = mfxU32(MFX_MIN(GetMaxBitrate(par) / 1000, UINT_MAX));
+                    par.calcParam.targetKbps = GetMaxBitrate(par) / 1000;
```

Comment:
is it really the same?

Reply:
```GetMaxBitrate``` returns ```mfxU32```, so I don't see a reason to crop it to ```UINT_MAX```